src/Nox.Generator/Application/Dto/EntityCreateDto.template.cs

```diff
@@ -13,6 +14,18 @@ namespace {{codeGeneratorState.ApplicationNameSpace }}.Dto;
 /// </summary>
 public partial class {{className}} : {{entity.Name}}UpdateDto
 {
-    // TODO Add Manual Keys to be set by the user
-
+{{- for key in entity.Keys }}
+    {{- if key.Type == "Nuid" || key.Type == "DatabaseNumber" || keyType == "Guid" -}}
```

Comment:
should be DatabaseGuid , Guid is not autogenerated

Reply:
Updated